.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: ['3.8']
-        toxenv: [sphinx11, sphinx14, docs, quality]
+        toxenv: [sphinx50, docs, quality]
```

Comment:
There's one more reference to `sphinx14`.
This might need to be updated as well.

https://github.com/openedx/edx-sphinx-theme/pull/199/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR42